### PR TITLE
change height button issue fixed

### LINKED ISSUE #4136: Timing Diagram- Decrease height and Increase height buttons are not working
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The Decrease height and Increase height buttons are not working in Timing Diagram Panel.

### Steps To Reproduce

1. Go to the Timing Diagram panel
2. Expand it 
3. Click on the Decrease and Increase height buttons. 

### Expected Behavior

1. The Increase height button should increase the height of the panel.
2. The Decrease height button should decrease the height of the panel.

Just like, the Increase size and Decrease size button works.

### Screenshots

image

### Device Information [optional]

```markdown
- OS: Windows 10
- Browser: Chrome
```

### Additional context

_No response_

### Are you working on this issue?

No

### PR DESCRIPTION
**Pull request by Jayendra-Welekar**
Fixes #4136

#### Describe the changes you have made in this PR -
    I tweaked the definition of " getFullHeight() "method to ensure that the height changes even if it gets the flag 
    count as 0.   The button was not working because flagCount was 0.
### Screenshots of the changes (If any) -
image
image

https://github.com/CircuitVerse/CircuitVerse/assets/114202520/7dff3880-2927-46e0-9128-5f449d5c4a71

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Review by niladrix719 — CHANGES_REQUESTED** [review]
use functions to avoid Duplication

**Review by Prerna-0202 — COMMENTED** [review]
Good work @Jayendra-Welekar, But I think that the recent changes you made have interchanged the button functionality. The "Increase height" button decreases the height, and vice versa. Please fix this. Also, remove the unnecessary lines of change (blanks).

File: simulator/src/plotArea.js [hunk 1]
```diff
@@ -214,22 +220,18 @@ const plotArea = {
         // Reset canvas
         this.clear();
         var ctx = this.ctx;
-
```
Comment (Prerna-0202) [inline simulator/src/plotArea.js:217]: Working is fine now 🎉. Just one question, why have you removed this blank line (there are 8-9 more lines you have removed)?"

Comment (Jayendra-Welekar) [reply simulator/src/plotArea.js:217]: sry for replying so late. Maybe it was some misunderstading from my side. I thought you asked to do so in your last request where you stated to change "unneccesary lines of changes(blanks)"

**Review by Prerna-0202 — APPROVED** [review]
Great 🎉

**Comment by Prerna-0202** [comment]
@vedant-jain03 @tachyons, we can merge this pr.

**Comment by Jayendra-Welekar** [comment]
hey @Prerna-0202 this is my first open source contribution so i am not sure if it is a problem but i am getting list and test as waiting for status to be reported

**Pull request merged**